### PR TITLE
fix tilesize

### PR DESCRIPTION
**Pull request by brccabral**
When a tileset has more than one object the drawing was displaying multiple objects.

### CODE REVIEW

File: example/raylib-tmx-example.c [hunk 1]
```diff
@@ -27,7 +27,7 @@ int main(int argc, char *argv[]) {
     InitWindow(screenWidth, screenHeight, "[raylib-tmx] example");
     SetTargetFPS(60);
 
-    tmx_map* map = LoadTMX("resources/desert.tmx");
+    tmx_map* map = LoadTMX(argv[1]);
```
Comment (RobLoach) [inline example/raylib-tmx-example.c:30]: Thanks! Unrelated change?
``` suggestion
    tmx_map* map = LoadTMX("resources/desert.tmx");
```

Comment (RobLoach) [inline example/raylib-tmx-example.c:30]: Thanks! Unrelated change?
``` suggestion
    tmx_map* map = LoadTMX("resources/desert.tmx");
```

Comment (brccabral) [reply example/raylib-tmx-example.c:30]: I wanted to test different tmx files and passing via command line was the easiest way, but I don't mind reverting it back

Comment (RobLoach) [reply example/raylib-tmx-example.c:30]: Good idea... Perhaps we check if a command line argument is passed, and use it if it's there. Otherwise, default to resources/desert.tmx :wink:

Comment (brccabral) [reply example/raylib-tmx-example.c:30]: Sure, that is great, but let's make a new PR then and revert this one for now

Comment (RobLoach) [reply example/raylib-tmx-example.c:30]: :shrug: Can have a follow up one :rocket:
